### DISCUSSION TITLE
Black background in dark mode iOS

### DISCUSSION THREAD

**Discussion by hmottestad** [community/community#55] [Mobile] [answered]
The black background in dark mode on iOS is not actually black, but instead very very dark grey. It would be great if it was truly black when using an iPhone with an oled screen.

**Comment by candyho** [comment, marked as answer]
Hey @hmottestad thanks for sharing your feedback! Touched base with our designers, we intentionally chose not to use pure black since it might result in pixel smudging. We're using #050505 which is optically black but bright enough to activate the pixels and avoid smudging.

**Reply by hmottestad** [reply]
Thanks for taking the time to answer. This makes a lot of sense!

**Reply by endolith** [reply]
Can you please make it pure black for OLED displays?

**Reply by hmottestad** [reply]
> Can you please make it pure black for OLED displays?

My understanding from the answer I got and a bit of research I did was that making it pure black makes scrolling an issue. 

When scrolling the OLED screen needs to quickly switch brightness of the three sub-pixels (red green blue). Switching the brightness is very quick as long as the sub-pixel is already ON. If the sub-pixel is completely OFF it takes a lot longer. This is something inherit to how OLED works, I don't quite understand exactly what's going on, but the result is that OLED has amazing response time for everything expect when switching from an OFF state. 

It might have gotten better in the last few years though. So it might be worth revisiting.
